### PR TITLE
docs: v0.3.4 changelog

### PR DESCRIPTION
**Pull request by craigpastro**
## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [ ] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.3.4] - 2023-02-02
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.3...v0.3.4)
+
+### Fixed
+
+* The new environment variables should work this time (#498)
```
Comment (rhamzeh) [inline CHANGELOG.md:17]:
```suggestion
* fix: using the new clearer env variables (#498). For the full list of env variables, see [.config-schema.json](https://github.com/openfga/openfga/blob/main/.config-schema.json).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,18 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
+
+### Removed
+
+* The ReadTuples endpoint has been removed. Please use Read with no tuple key instead. (#495)
```
Comment (rhamzeh) [inline CHANGELOG.md:21]:
```suggestion
* The ReadTuples endpoint has been removed. Please use [Read](https://openfga.dev/api/service#/Relationship%20Tuples/Read) with no tuple key instead. (#495)
 e.g. `POST /stores/<store_id>/read` with `{}` as the body
```

**Review by rhamzeh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,19 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.3...v0.3.4)
+
+### Fixed
+
+* Fixed environment variable mapping (#498). For the full list of environment variables see [.config-schema.json](https://github.com/openfga/openfga/blob/main/.config-schema.json).
+
+### Removed
+
+* The ReadTuples endpoint has been removed. Please use [Read](https://openfga.dev/api/service#/Relationship%20Tuples/Read) with no tuple key instead. (#495)
```
Comment (jon-whit) [inline CHANGELOG.md:21]: Can we put the 'e.g.' bit on a newline so it renders more nicely.

* blah...

  e.g. `POST /stores/<store_id>/read` with `{}` as the body

**Review by jon-whit — DISMISSED** [review]

**Review by rhamzeh — DISMISSED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,22 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.3...v0.3.4)
+
+### Fixed
+
+* Fixed the environment variable mapping (#498). For the full list of environment variables see [.config-schema.json](https://github.com/openfga/openfga/blob/main/.config-schema.json).
+
+### Added
+
+* Added OpenTelemetry tracing (#499)
```
Comment (jon-whit) [inline CHANGELOG.md:21]: Let's add a quick follow up PR to openfga/openfga.dev that documents how to wire up tracing (including the flags and docker compose if that's what you're using).

**Review by jon-whit — APPROVED** [review]

**Review by jon-whit — APPROVED** [review]

**Pull request merged**